src/hooks/useReactQuery.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import { useState } from 'react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+
+export default function ReactQueryProviders({ children }: React.PropsWithChildren) {
```

Comment:
저희 interface로 타입 명시해주기로 했던 것으로 기억합니다!

```suggestion
interface ReactQueryProvidersProps = {
  children: React.ReactNode;
};

export default function ReactQueryProviders({ children }: ReactQueryProvidersProps) {
```

Reply:
✅ f1aa2b3ea7f059ba8dbab821d3f15ccbe5d57510